nextstrain/auspice
# 1055 export selected tips

### Description of proposed changes    
Adds the ability to export only tips which are selected from the download data modal. Refreshes layout of download area of modal, and adds descriptions of the data available for download.

### Testing
- Make sure each old data download option still works correctly
- Make sure the new option, selected data, correctly downloads only tips which are selected (visible and in view on tree graph)
- Make sure each data download text block has the right description
- Make sure numbers of authors, total strains, selected strains report correctly in the modal

### Thank you for contributing to Nextstrain!
